pimd/pim6_cmd.c

```diff
@@ -448,6 +451,424 @@ DEFPY (no_ipv6_pim_rp_prefix_list,
 	return pim_process_no_rp_plist_cmd(vty, rp_str, plist);
 }
 
+DEFPY (show_ipv6_pim_channel,
+       show_ipv6_pim_channel_cmd,
+       "show ipv6 pim [vrf NAME] channel [json$json]",
+       SHOW_STR
+       IPV6_STR
+       PIM_STR
+       VRF_CMD_HELP_STR
+       "PIM downstream channel info\n"
+       JSON_STR)
+{
+	struct vrf *v;
+	bool uj = !!json;
+
+	v = vrf_lookup_by_name(vrf ? vrf : VRF_DEFAULT_NAME);
+
+	if (!v)
+		return CMD_WARNING;
+
+	pim_show_channel(v->info, vty, uj);
+
+	return CMD_SUCCESS;
+}
+
+DEFPY (show_ipv6_pim_ssm_range,
+       show_ipv6_pim_ssm_range_cmd,
+       "show ipv6 pim [vrf NAME] group-type [json$json]",
+       SHOW_STR
+       IPV6_STR
+       PIM_STR
+       VRF_CMD_HELP_STR
+       "PIM group type\n"
+       JSON_STR)
+{
+	struct vrf *v;
+	bool uj = !!json;
+
+	v = vrf_lookup_by_name(vrf ? vrf : VRF_DEFAULT_NAME);
+
+	if (!v)
+		return CMD_WARNING;
+
+	ip_pim_ssm_show_group_range(v->info, vty, uj);
+
+	return CMD_SUCCESS;
+}
+
+DEFPY (show_ipv6_pim_interface,
+       show_ipv6_pim_interface_cmd,
+       "show ipv6 pim [mlag] [vrf NAME] interface [detail$detail|WORD$word] [json$json]",
```

Comment:
Since there's no MLAG support for IPv6 currently, we should remove this and add it back if/when IPv6 MLAG support is added. Having "dead" CLI is confusing to the user otherwise.

(Repeated below)

Reply:
Fixed